client-projects/the-lumery/whoWeWorkWithFilter.ts

```diff
@@ -0,0 +1,120 @@
+import { CLIENT_FILTER_BUTTON_SELECTOR } from "./selectors";
+import { prefersReducedMotion } from "./utils";
+
+type ClientCategory =
+  | "customers"
+  | "technology-vendors"
+  | "industry-associations";
+
+export const createClientFilterButtons = () => {
```

Comment:
The code is good, and I was able to see what this is all about. But for the sake of the next developer, maybe you could add a comment here saying what the point of this is? Which I believe is:

Webflow doesn't let us create `<button>` elements, so this function grabs the `<a>` elements which *should* be `<button>`s and converts them.

Reply:
ah yep good call, will do